lib/extract-lmdb-vault.js

```diff
@@ -0,0 +1,2 @@
+module.exports = {
```

Comment:
Is this left over from some other branch? I'm having trouble understanding what this does.

Reply:
Wups, good catch, this is redundant.